internal/cmd/run/run.go

```diff
@@ -136,6 +136,8 @@ func Command() *cobra.Command {
 	sc.StringToString("experiment", "sauce::experiment", map[string]string{}, "Specifies a list of experimental flags and values")
 	sc.Bool("dry-run", "dryRun", false, "Simulate a test run without actually running any tests.")
 	sc.Int("retries", "sauce::retries", 0, "Retries specifies the number of times to retry a failed suite (sauce mode only)")
+	sc.Int("maxAttempt", "sauce::maxAttempt", 1, "Specifies the maximum number of attempts to run a suite (sauce mode only)")
+	sc.Int("minPass", "sauce::minPass", 0, "Specifies the threshold of passed attempts (sauce mode only)")
```

Comment:
[Duplicate] Shouldn't this value to be `1` ?
Otherwise that means that the suite can pass when no attempts actually passed.